src/index.html

```diff
@@ -6,9 +6,63 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <div class="container">
+
+      <header class="header">
+        <a class="header__logo" href="index.html">
+          <img
+            class="header__logo-img"
+            src="images/logo.png"
+            alt="logo"
+          >
+        </a>
+        <nav class="header__nav">
+          <ul class="header__nav-list">
+            <li class="header__nav-item">
+              <a
+                class="header__nav-link is-active"
+                href="#"
+              >
+                Apple
+              </a>
+            </li>
+            <li class="header__nav-item">
+              <a class="header__nav-link" href="#">Samsung</a>
+            </li>
+            <li class="header__nav-item">
+              <a class="header__nav-link" href="#">Smartphones</a>
+            </li>
```

Comment:
```suggestion
          <ul class="header__nav-list">
            <li class="header__nav-item">
              <a
                class="header__nav-link is-active"
                href="#"
              >
                Apple
              </a>
            </li>
            
            <li class="header__nav-item">
              <a class="header__nav-link" href="#">Samsung</a>
            </li>
            
            <li class="header__nav-item">
              <a class="header__nav-link" href="#">Smartphones</a>
            </li>
            
```
add empty blanks between siblings for better readability (the previous comment)